main.go

```diff
@@ -15,7 +15,7 @@ func main() {
 	logger := log.Default()
 
 	// * Config
-	config, err := config.Load(CONFIG_FILE)
+	config, err := config.Load(CONFIG_FILE, DUMP_MIGRATIONS, logger)
 	ehp(err, logger)
```

Comment:
we can also change the ehp to be a method instead of a func which has a struct dep logger (should we make that the default way of doing these?)

Reply:
I really don't think this is the way to go about it. It'll just force us to create meaningless structs for everything with the only purpose of holding dependencies. Eventually some kind of injection could make more sense.